magefile.go

```diff
@@ -305,3 +306,34 @@ func Vuln() error {
 func installGoVulnIfNotPresent() error {
 	return installIfNotPresent("govulncheck", "golang.org/x/vuln/cmd/govulncheck@latest")
 }
+
+func installGoMobileIfNotPresent() error {
+	return installIfNotPresent(gomobile, "golang.org/x/mobile/cmd/gomobile@latest")
+}
+
+// IOS Generates the iOS packages
+// Note: this command also installs "gomobile" if not present
+func IOS() error {
```

Comment:
Should this target, as well as the Android target, be documented in a README?

Reply:
yes good call